overlays/build-tools-in-build-inputs.nix

```diff
@@ -9,10 +9,109 @@ let
   inherit (import ../lib { inherit lib; }) checkMkDerivationFor;
 
   buildTools = [
+    "antlr"
+    "appstream-glib"
```

Comment:
Do we want to keep that for now or add a comment that we want to change it soon?

Reply:
I would remove it for now. Maybe we should even recommend people not to add it since there is not much benefit of running appstream validation downstream.